_tests/apps_test.go

```diff
@@ -14,37 +14,34 @@ var _ = Describe("Apps", func() {
 		})
 
 		It("can't get app info", func() {
-			output, err := execute("deis info -a %s", app1Name)
-			Expect(err).To(HaveOccurred())
-			Expect(output).To(ContainSubstring("NOT FOUND"))
+			Expect(execute("deis info -a %s", app1Name)).To(BeASuccessfulCmdWithOutput(ContainSubstring("NOT FOUND")))
 		})
 
 		It("can't get app logs", func() {
-			output, err := execute("deis logs -a %s", app1Name)
+			out, err := execute("deis logs -a %s", app1Name)
 			Expect(err).To(HaveOccurred())
-			Expect(output).To(ContainSubstring("NOT FOUND"))
+			Expect(out).To(ContainSubstring("NOT FOUND"))
 		})
 
 		// TODO: this currently returns "Error: json: cannot unmarshal object into Go value of type []interface {}"
 		XIt("can't run a command in the app environment", func() {
-			output, err := execute("deis apps:run echo Hello, 世界")
+			out, err := execute("deis apps:run echo Hello, 世界")
 			Expect(err).To(HaveOccurred())
-			Expect(output).To(ContainSubstring("NOT FOUND"))
+			Expect(out).To(ContainSubstring("NOT FOUND"))
 		})
 
 		It("can create an app", func() {
-			output, err := execute("deis apps:create %s", app1Name)
-			Expect(err).NotTo(HaveOccurred())
-			Expect(output).To(SatisfyAll(
+			Expect(execute("deis apps:create %s", app1Name)).To(BeASuccessfulCmdWithOutput(
 				ContainSubstring("Creating Application... done, created %s", app1Name),
 				ContainSubstring("Git remote deis added"),
-				ContainSubstring("remote available at ")))
-			output, err = execute("deis apps:destroy --confirm=%s", app1Name)
-			Expect(err).NotTo(HaveOccurred())
```

Comment:
Shouldn't we still be checking if there's an error running these commands? The new commands just seem to check that the output is what's expected. What about the command exiting non-zero?